includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -186,17 +192,55 @@ public function getUserUpvotes( $userId ) {
 	}
 
 	/**
+	 * Get new upvotes for revisions made by given user
+	 *
+	 * @param int $userId
+	 * @return bool|array
+	 */
+	public function getUserNewUpvotes( $userId ) {
+		$status = $this->getUserUpvotesStatus( $userId );
+
+		if ( $status['notified'] ) {
+			return [];
+		}
+
+		$upvotes = $this->getUserUpvotes( $userId, $status['last_notified'] );
+
+		return $upvotes;
+	}
+
+	private function getUserUpvotesStatus( $userId ) {
+		$db = $this->getDatabaseForRead();
+
+		$status = ( new \WikiaSQL() )
+			->SELECT_ALL()
+			->FROM( self::UPVOTE_USERS_TABLE )
+			->WHERE( 'user_id' )->EQUAL_TO( $userId )
+			->runLoop( $db, function( &$status, $row ) {
+				$status = [
+					'total' => (int) $row->total,
+					'new' => (int) $row->new,
```

Comment:
Yes, the question is whether we are using those fields in the code invoking this method?